mypy/semanal.py

```diff
@@ -2356,6 +2357,64 @@ def is_classvar(self, typ: Type) -> bool:
     def fail_invalid_classvar(self, context: Context) -> None:
         self.fail('ClassVar can only be used for assignments in class body', context)
 
+    def process_module_assignment(self, s: AssignmentStmt) -> None:
```

Comment:
Why do you need a function that just unconditionally calls another function? Maybe one is enough?